core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -1,45 +1,139 @@
 <div class="oppia-opportunities-list-item protractor-test-opportunity-list-item">
   <div class="oppia-opportunities-details">
     <div class="oppia-opportunities-subheading oppia-animate-fadein">
-      <span title="<[$ctrl.opportunity.subheading]>"
+      <span title="{{ opportunity?.subheading }}"
```

Comment:
Why do we need the question mark?

Reply:
As part of the loading animation, this component is rendered without an opportunity; the `?` allows the component to render when `opportunity` is undefined.